src/static_data/get_static_data.go

```diff
@@ -0,0 +1,236 @@
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2030.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2045.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2100.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2115.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2130.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2145.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2200.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2215.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2230.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2245.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2300.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2315.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2330.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211228.2345.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211229.0000.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211229.0015.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211229.0030.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211229.0045.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211229.0100.bz2",
+		},
+	}
+
+	bgpnyfiles := Folder{
+		FolderName: "bgpnyfiles",
+		Urls: []string{
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211201.0000.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211201.0015.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211201.0030.bz2",
+			"http://routeviews.org/route-views.ny/bgpdata/2021.12/UPDATES/updates.20211201.0045.bz2",
+		},
+	}
+
+	bgpupdatefiles := Folder{
+		FolderName: "bgpupdatefiles",
+		Urls: []string{
+			"http://routeviews.org/route-views.chicago/bgpdata/2016.06/UPDATES/updates.20160629.1330.bz2",
+			"http://routeviews.org/route-views.chicago/bgpdata/2016.06/UPDATES/updates.20160629.1345.bz2",
+			"http://routeviews.org/route-views.chicago/bgpdata/2016.06/UPDATES/updates.20160629.1400.bz2",
+			"http://routeviews.org/route-views.chicago/bgpdata/2016.06/UPDATES/updates.20160629.1415.bz2",
+			"http://routeviews.org/route-views.chicago/bgpdata/2016.06/UPDATES/updates.20160629.1430.bz2",
+			"http://routeviews.org/route-views.chicago/bgpdata/2016.06/UPDATES/updates.20160629.1445.bz2",
+		},
+	}
+
+	// Download folders
+	err := downloadFolder(bgptest1)
```

Comment:
create a list and of folders and pass it to a function which will iterate over and do what you are expecting.

Reply:
for _, dir := range listOfFolder {

`for _, dir := range listOfFolder {
      downloadFolder(dir)
}`